eks-deployments/patch-coredns/Patch-Coredns.ps1

```diff
@@ -55,9 +55,20 @@ if (!(Get-Command -Name 'kubectl' -ErrorAction:SilentlyContinue)) {
     throw 'kubectl not found'
 }
 
+[string]$clusterVersionInfo = Invoke-Expression -Command "kubectl version --short=true -o json" 
```

Comment:
Can you make these lines coherent with the rest of the code here: https://github.com/aws-samples/amazon-eks-gmsa/blob/master/eks-deployments/webhook/patch-webhook-template.ps1#L52-#L55

Reply:
I was referring to that code section when implementing the changes. It only takes the minor version into account, so we can assume that is good enough in our context, right?